### PR TITLE
Changing default branch for github actions

### PR DESCRIPTION
**Pull request by chaitanya-baraskar**
## Change Overview

Updating default branch for github actions

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [x] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -3,10 +3,11 @@ name: kanister
 on:
   push:
     branches:
-      - main
+      - master
   pull_request:
     branches: 
-      - main
+      - master
+
```
Comment (PrasadG193) [inline .github/workflows/main.yml:10]:
```suggestion
```

**Review by PrasadG193 — APPROVED** [review]
Please remove the extra new line.

**Comment by viveksinghggits** [comment]
hey @chaitanya-baraskar 
what does `minio test` and `kind test` mean? From the names it looks like one run against `minio` and another runs on `kind` is that correct?

**Review by viveksinghggits — APPROVED** [review]

**Comment by PrasadG193** [comment]
PR needs to be merged manually!
image

**Pull request merged**